src/StructuredLogViewer.Avalonia/StructuredLogViewer.Avalonia.csproj

```diff
@@ -29,6 +29,8 @@
     <PackageReference Include="Avalonia.Diagnostics" Version="0.10.3" Condition=" '$(Configuration)' == 'Debug' " />
     <PackageReference Include="Avalonia.AvaloniaEdit" Version="0.10.0" />
     <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+    <PackageReference Include="SkiaSharp" Version="2.88.0-preview.145" />
+    <PackageReference Include="HarfBuzzSharp" Version="2.8.2-preview.140" />

```

Comment:
why do you need these additional package references?

Reply:
At the time I wrote it the transitive references were not new enough to contain the osx-arm64 native assets.

Reply:
Might be worth a comment :)